Projects/PaywallTester/Project.swift

```diff
@@ -0,0 +1,83 @@
+import Foundation
+import ProjectDescription
+import ProjectDescriptionHelpers
+
+let allDestinations: Destinations = [
+    .iPhone,
+    .iPad,
+    .mac,
+    .macWithiPadDesign,
+    .macCatalyst,
+    .appleWatch,
+    .appleTv,
+    .appleVision,
+    .appleVisionWithiPadDesign
+]
+
+let allDeploymentTargets: DeploymentTargets = .multiplatform(
+    iOS: "15.0",
+    macOS: "11.0",
+    watchOS: "7.0",
+    tvOS: "14.0"
```

Comment:
These seem slightly different to the ones currently in the project?
<img width="415" alt="image" src="https://github.com/user-attachments/assets/cb10e805-2044-4145-936c-c54b500f37c9" />